roles/datastore/datastore_setup/tasks/update_crawler.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: "Update source data with crawler"
```

Comment:
This command is run by `cron`, so isn't totally clear why it's also a separate task.

Suggest adding a comment to explain why it has to be here too.